docs/core/reports/customisation.md

```diff
@@ -83,6 +83,108 @@ Then this will be displayed at the top of reports:
 
 Note that you can also specify a path to a config file using `-c`.
 
+## Listing software versions
+
+For modules where version information can be extracted from log files,
+or when not available in log files, you can alternatively add them in
+the configuration. Software versions can be specified in YAML format
+using two options.
+
+### Option 1: Grouping software sersions by process name
+
+For each process, software versions can be organized within a nested YAML
+dictionary with three levels: `process` -> `software` -> `version(s)`.
```

Comment:
I think we should avoid the term `process` as that's a Nextflow thing, not used anywhere else in MultiQC.

Better to call it `group` maybe?

Reply:
Yes, `process` might be mimicking Nextflow to much. Let's go for `group`.